src/shared-components/optionButton/index.tsx

```diff
@@ -4,17 +4,15 @@ import PropTypes from 'prop-types';
 import Style from './style';
 import { CheckmarkIcon } from '../../icons';
 import { isDefined } from '../../utils/isDefined';
+import { ThemeType } from '../../constants';
 
 export interface OptionButtonProps {
-  borderRadius?: string;
+  borderRadius?: valueof<ThemeType['BORDER_RADIUS']>;
   buttonType?: 'primary' | 'secondary';
   /**
    * Show custom icon in the unselected state
    */
   icon?: React.ReactNode;
-  /**
-   * Label
```

Comment:
This appears to have been accidentally committed as it makes no sense in the storybook docs page for `onClick`.